### PR TITLE
Linux build version script to hide exports of dependencies

### PR DESCRIPTION
**Pull request by Gillibald**
**Description of Change**

Currently symbols of Freetype etc. are exported during the linker process. This leads to errors when used with applications that load build in third party libraries.

**Bugs Fixed**

- Related to issue #927

### CODE REVIEW

File: cake/BuildExternals.cake [hunk 1]
```diff
@@ -596,7 +596,7 @@ Task ("externals-linux")
             $"skia_use_system_expat=false skia_use_system_freetype2=false skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false " +
             $"skia_enable_gpu={(SUPPORT_GPU ? "true" : "false")} " +
             $"extra_cflags=[ '-DSKIA_C_DLL' ] " +
-            $"extra_ldflags=[ '-static-libstdc++', '-static-libgcc' ] " +
+            $"extra_ldflags=[ '-static-libstdc++', '-static-libgcc', '-Wl,--version-script={ROOT_PATH.CombineWithFilePath("native-builds/libHarfBuzzSharp_linux/libSkiaSharp.map")}' ] " +
```
Comment (mattleibow) [inline cake/BuildExternals.cake]: This is the libHarfBuzzSharp path.

```suggestion
            $"extra_ldflags=[ '-static-libstdc++', '-static-libgcc', '-Wl,--version-script={ROOT_PATH.CombineWithFilePath("native-builds/libSkiaSharp_linux/libSkiaSharp.map")}' ] " +
```

**Comment by mattleibow** [comment]
I see an error:
> /usr/bin/ld:/__w/1/s/native-builds/libSkiaSharp_linux/libSkiaSharp.map:7: syntax error in VERSION script

**Comment by Gillibald** [comment]
Let me fire up a VM quick

**Comment by mattleibow** [comment]
I think it was missing the last semicolon... Let's see.

**Comment by mattleibow** [comment]
Do you use a VM of WSL?

**Comment by mattleibow** [comment]
Seems to have passed the build of libSkiaSharp for Linux.

**Comment by Gillibald** [comment]
Y same on my machine. Looks okay now.

**Comment by Gillibald** [comment]
I wonder why Fontconfig is still exported. There seems to be a version script for glibc as well.

**Pull request merged**